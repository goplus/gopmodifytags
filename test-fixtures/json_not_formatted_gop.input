type x struct {
    Foo int
    bar int
}
type y struct {
    Foo int
    bar int
}
