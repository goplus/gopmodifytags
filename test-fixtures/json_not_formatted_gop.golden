{
  "start": 2,
  "end": 3,
  "lines": [
    "\tFoo int `json:\"foo\"`",
    "\tbar int `json:\"bar\"`"
  ]
}